addons/account/views/account_portal_templates.xml

```diff
@@ -18,6 +18,9 @@
                 <t t-set="title">Invoices &amp; Bills</t>
                 <t t-set="url" t-value="'/my/invoices'"/>
                 <t t-set="placeholder_count" t-value="'invoice_count'"/>
+                <t t-set="content" t-value="[
+                    {'text': 'Invoices',
+                     'counter': 'invoice_count'}]"/>
```

Comment:
Somehow having a "title" parameter, a "placeholder_count" parameter and then this content thing is creating duplicates.
I'm thinking stick to the old parameters if possible.

I see that in some usages you pass multiple text and counters, creating such a generic structure seems a bit overkill to build a card item.
Either stick with a title and if it overflows, it goes back to next row, or introduce a "description" variable that allows adding text below the title, or something like that.
Goal is to keep it simple.

Also, I don't think specifying text like you're doing here is working with translations.
You could check by exporting the .pot file (don't hesitate to ask if you don't know how to do it).